src/js/dataMapping/search/awardType.js

```diff
@@ -3,26 +3,21 @@
   * Created by Kevin Li 11/4/16
   **/
 
+import { longTypeDescriptionsByAwardTypes } from "../../dataMapping/awardsv2/longAwardTypeDescriptions";
+
 /* eslint-disable quote-props */
 // disable quote-props for consistency sake (we need leading zeroes)
 export const awardTypeCodes = {
-    'A': 'Blanket Purchase Agreements (BPA) Calls',
-    'B': 'Purchase Orders (PO)',
-    'C': 'Delivery Orders (DO)',
-    'D': 'Definitive Contracts',
-    'E': 'Unknown Type',
-    'F': 'Cooperative Agreement',
-    'G': 'Grant for Research',
-    'S': 'Funded Space Act Agreement',
-    'T': 'Training Grant',
-    'IDV_A': 'Government-Wide Acquisition Contract (GWAC)',
```

Comment:
Just want to confirm that we want to change 
`'IDV_A': 'Government-Wide Acquisition Contract (GWAC)',	
'IDV_B': 'Multi-Agency Contract, Other Indefinite Delivery Contract (IDC)',
'IDV_C': 'Federal Supply Schedule (FSS)',	
'IDV_D': 'Basic Ordering Agreement (BOA)',	
'IDV_E': 'Blanket Purchase Agreements (BPA)'` in `awardTypeCodes` to their respective versions in `longTypeDescriptionsByAwardTypes `

Reply:
And if so, we might want to consider changing `idvAwardTypes` in `dataMapping/contracts/idvAwardtypes.js`.

Reply:
@jonhill13 thanks for catching this, we don't want to change those at all.